compose.yml

```
@@ -73,7 +52,131 @@ services:
+  sodo-search:
+    <<: *service-template
+    working_dir: /home/ghost/apps/sodo-search
+    command: [ "yarn", "dev" ]
+    ports:
+      - "4178:4178"
+    environment:
+      - NX_DAEMON=false
+    profiles: [ sodo-search ]
+
+  comments-ui:
+    <<: *ghost-base
```

YAMLlint (1.35.1)

[error] found undeclared alias "ghost-base"

(anchors)